### ISSUE TITLE
Cost constraints

### ISSUE THREAD

**Issue by morvael** [BattleScribe/Release#4]
Please check the attached test case.

I have made two entries, that cost 0 points, with an option that costs 50 points in a 100 point roster.
One entry ("Entry Max") has the following constraints:
```
max 33% pts limit in Roster
max 33 pts in Parent
```
Second entry ("Entry Min") has the following constraints:
```
min 33% pts limit in Roster
min 33 pts in Parent
```
In all cases "and all child selections" is selected.

In the roster editor I see the following messages:
... must spend 33pts more of Entry Min (minimum 33pts / 33% of pts limit)
... must spend 33pts more of Entry Min (minimum 33pts)

The messages show for Entry Min, whereas they should trigger for Entry Max. It looks like the constraint is unable to determine the cost of parent, despite correct cost being shown in the roster selection tree.

I tried alternate methods which increment min/max pts in Parent by 1 for every 3 pts of roster limit, and they are broken in the same way - correct min/max is estabilished, but the constraint thinks the cost of parent is 0.

Test case
[link](https://drive.google.com/open?id=1TQEakTBPbw5XMET-BI90aHVlYXnxpHeT)

**Comment by Jonskichov** [comment]
Fixed in 2.01.11.104 on desktop

Android and iOS builds coming in the next day or so

**Comment by Jonskichov** [comment]
Fixed in 2.01.17.270 on Android

**Comment by Jonskichov** [comment]
Fixed in 2.01.15.113 on iOS
